app/user.go

```diff
@@ -516,11 +516,11 @@ func (a *App) GetUsersNotInTeamEtag(teamId string, restrictionsHash string) stri
 }
 
 func (a *App) GetUsersInChannel(channelId string, offset int, limit int) ([]*model.User, *model.AppError) {
-	result := <-a.Srv.Store.User().GetProfilesInChannel(channelId, offset, limit)
-	if result.Err != nil {
-		return nil, result.Err
+	result, err := a.Srv.Store.User().GetProfilesInChannel(channelId, offset, limit)
```

Comment:
You can just `return a.Srv.Store.User().GetProfilesInChannel(channelId, offset, limit)` here